weather_dl/download_pipeline/util.py

```diff
@@ -58,3 +62,16 @@ def ichunked(iterable: t.Iterable, n: int) -> t.Iterator[t.Iterable]:
             yield itertools.chain([first], it)
     except StopIteration:
         pass
+
+
+def copy(src: str, dst: str) -> None:
+    """Copy data via `gcloud storage cp`.
+
+    Here, we take advantage of GCP's faster storage transfer:
+    https://cloud.google.com/blog/products/storage-data-transfer/new-gcloud-storage-enables-super-fast-data-transfers/
+    """
+    try:
+        subprocess.run(f'gcloud alpha storage cp {src!r} {dst!r}', shell=True, check=True, capture_output=True)
```

Comment:
Generally it's best to avoid `shell=True` for [security considerations](https://docs.python.org/3/library/subprocess.html#security-considerations):
```suggestion
        subprocess.run(['gcloud', 'alpha', 'storage', 'cp', src, dest], check=True, capture_output=True)
```